Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,84 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+            
+            char []equal = {'='};
+            string []arr=equation.Split(equal);
+            string right=arr[1];
+            char []multi = {'*'};
+            int a=0,b=0,c=0;
+            string []left=arr[0].Split(multi);
+            string l1=left[0];
+            string l2=left[1];
+            for(int i=0;i<l1.Length;i++){

```

Comment:
To check if string contains a char use string.Contains() method, rather than iterating through the string.
C# as a language has these common methods out of the box, so first look if there is any method available, rather than writing your own, but do have a look how that has been implemented internally, so you have a better understanding of the language.